src/canister/individual_user_template/src/data_model/mod.rs

```diff
@@ -58,7 +67,9 @@ pub struct CanisterData {
     #[serde(default)]
     pub session_type: Option<SessionType>,
     #[serde(default)]
-    pub last_access_time: Option<SystemTime>
+    pub last_access_time: Option<SystemTime>,
+    #[serde(default)]
+    pub last_canister_functionality_access_time: Option<SystemTime>,
```

Comment:
Why can't we use `last_access_time`. It was added for same purpose.

Reply:
wanted to differentiate between last_user_access_time and last_canister_access_time